gc/base/ScanClassesMode.hpp

```diff
@@ -36,23 +36,57 @@ class MM_ScanClassesMode : public MM_Base
 	volatile uintptr_t _scanClassesMode;
 
 public:
+	typedef enum {
```

Comment:
ok.  Are these future commits going to be under the same Issue as this or a different one?  If you plan on using the current Issue then you should change this commit message to remove the fix and just say issue